apps/src/templates/AgeDialog.jsx

```diff
@@ -50,7 +47,21 @@ const styles = {
   },
 };
 
-const sessionStorageKey = 'anon_over13';
+/*
+ * SignInOrAgeDialog uses 'anon_over13' as its session storage key.
+ * We want users seeing that dialog to have to input their age, so using
+ * a different session storage key here.
+ */
+const AGE_DIALOG_SESSION_KEY = 'ad_anon_over13';
+
+export const signedOutOver13 = () => {
+  const val = sessionStorage.getItem(AGE_DIALOG_SESSION_KEY);
+  if (val) {
```

Comment:
Nit: We could condense this conditional to `return val && val === 'true'`.

Reply:
Or maybe just `val === 'true'`?

Reply:
yes! good call